### ISSUE TITLE
Synoptic panel state not loading correctly

### ISSUE THREAD

**Issue by YuaFox** [SCADA-LTS/Scada-LTS#2974]
**Describe the bug**
I have the following state:
![image](https://github.com/user-attachments/assets/b7396ef2-1fc8-49b1-969f-e3b9cf1adcab)

If im in the panel and i change the value to 0, the state color is red.
When i refresh the window it shows as green, it should be red.

**To Reproduce**
1. Go to the panel
2. Chnage the value from 1->0
3. The color is changed to red
4. Refresh
5. (Bug) The color is green, should be red

**Expected behavior**
State should evaluate the condition when loading the page but it always marks it as positive (green color).

**Screenshots**
State when value changes to 0
![image](https://github.com/user-attachments/assets/d446cd30-5431-48dc-821b-b875111058b5)
After refreshing the tab
![image](https://github.com/user-attachments/assets/1a631ffa-f673-4ec4-897d-41adcae44a3c)

**Desktop (please complete the following information):**
 - OS: Linux
 - Browser Chrome 126.0.6478.182 (64-bit)

**Comment by YuaFox** [comment]
When the synoptic panel is loaded onPointEnabledUpdate and onPointValueUpdate are executed.
![image](https://github.com/user-attachments/assets/b3fec342-8bb2-44c7-8857-704ad1afbfaf)

```
onPointEnabledUpdate(enabled) {
    if(enabled) {
       ....
        // This should check what color to use
        this.changeComponentColor(`${this.componentId}`, this.enabledColor); 
    } else {
        ....
    }
},
```


There's a race between the 2 functions:
- onPointEnabledUpdate always use the enabled color
- onPointValueUpdate checks the condition before using enabled/disabled color

onPointEnabledUpdate should check what color to use when enabling

**Comment by Limraj** [comment]
Hi @YuaFox,
Synoptic panels are an interesting solution and are worth investing in. Thanks you.

Propose issue:
#2978 Added ability set value on Synoptic Panel for Data Point settable;

Other issue suggestions:
#2805 Upgrade Node;
#2976 Fixed refresh chart-line and chart-range;

**However, if these issues do not suit your preferences, we are open to your further suggestions for improvements.**

We also encourage you to report any problems you notice, even if you do not intend to address them.

Regards,
Kamil Jarmusik

**Comment by Limraj** [comment]
Hi @YuaFox,
we have two more suggestions:
#2955 - You should upgrade the versions according to what dependabot suggests. However, this can be troublesome, mainly due to the lack of backward compatibility, the requirement for a newer Node, etc. So it may not be possible to achieve it in a reasonable time, if there are a lot of problems, we leave it for now;
#2957;

Regards,
Kamil Jarmusik

**Comment by YuaFox** [comment]
Hi @Limraj 

These suggestions are interesting and will improve the quality of the code, I will review them and see what I can do.

Another improvement that would be interesting would be some kind of interactivity, e.g. actuation buttons. Combined with node-red many interesting things could be done.

Node-red is a widely used plcs/iot automation software.

https://nodered.org/

**Comment by Limraj** [comment]
@YuaFox, Unfortunately, the constant introduction of revolutions has led us to a place where the new UI is a burden for us. Rather, we try to improve what is there by minimizing the number of changes. However, you can create an issue on this topic(node-red), describe how we can use it, with example, with links to documentation, it will be something for us to consider.

**Comment by Limraj** [comment]
https://github.com/SCADA-LTS/Scada-LTS/pull/2979
Here you still have something to improve, it is worth trying to finish one issue before starting the next one. Unless the task is too complex, perhaps poorly defined, too general, etc. Then you can also propose an new issue for a given issue, with a more complete description and a realistic scope of work.
